iina/KeyCodeHelper.swift

```diff
@@ -7,7 +7,7 @@
 //
 
 import Foundation
-
+import Carbon
```

Comment:
Ouch…is there no other way to do this? I'd really not like us to use deprecated APIs.

Reply:
Unfortunately no…… `UCKeyTranslate` itself is not from Carbon, but in order to get the "key layout data", some HIToolbox APIs are needed.

Reply:
I guess…